src/index.html

```diff
@@ -6,8 +6,46 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&family=Inter:wght@400;800&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        class="card__product-image"
+        src="./images/imac.jpeg"
+        alt="Imac"
+      >
+
+      <h1 class="card__product-title">
+        APPLE A1419 iMac 27" Retina
+        <br>
+        5K Monoblock (MNED2UA/A)
+      </h1>
+
+      <div class="card__product-code">
+        Product code: 195434
+      </div>
+
+      <div class="card__rating">
+        <div class="card__rating-stars">
+          <div class="card__rating-stars-star card__rating-stars-star--active"></div>
```

Comment:
use solution from previous `Stars` task. Calculate using modifier 
```suggestion
          <div class="card__rating-stars-star"></div>
```